integrations/java/iceberg-1.5/openhouse-java-runtime/build.gradle

```diff
@@ -19,7 +19,7 @@ configurations {
 }
 
 ext {
-  icebergVersion = '1.5.2'
+  icebergVersion = '1.5.2.1'
```

Comment:
Same comment of getting the iceberg version 1.5.x from a common place.

Reply:
Done.